client/my-sites/checkout/composite-checkout/wpcom/components/wp-checkout-order-summary.js

```diff
@@ -184,7 +184,7 @@ function getPlanFeatures( plan, translate, hasDomainsInCart ) {
 		return [
 			! hasDomainsInCart && translate( 'Free domain for one year' ),
 			translate( 'Unlimited access to our library of Premium Themes' ),
-			translate( 'Subscriber-only content and simple payment buttons' ),
+			translate( 'Subscriber-only content and payment buttons' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.